### PR TITLE
fix(molgenis-components): Save changed default value in new record

### PR DESCRIPTION
**Pull request by MaxPostema**
The computed code resets the changed default value back to the initial default value ( not reflected in the form ).
This PR makes you able to change a default value, save, and have it really be the value you have chosen.

fixes: https://github.com/molgenis/molgenis-emx2/issues/2840

### CODE REVIEW

File: apps/molgenis-components/src/components/forms/RowEdit.vue [hunk 1]
```diff
@@ -86,11 +86,6 @@ export default {
       type: Array,
       required: false,
     },
-    // defaultValue: when creating new record, this is initialization value
```
Comment (mswertz) [inline apps/molgenis-components/src/components/forms/RowEdit.vue:89]: are you sure this is never used? this is a way to set default value via another route then via the metadata.

File: apps/molgenis-components/src/components/forms/RowEdit.vue [hunk 2]
```diff
@@ -263,9 +256,11 @@ export default {
     },
   },
   created() {
-    if (this.defaultValue) {
```
Comment (mswertz) [inline apps/molgenis-components/src/components/forms/RowEdit.vue:266]: idem

File: apps/molgenis-components/src/components/forms/RowEdit.vue [hunk 3]
```diff
@@ -263,9 +256,11 @@ export default {
...
-      this.internalValues = deepClone(this.defaultValue);
-    }
+    this.tableMetaData.columns.forEach((column: IColumn) => {
```
Comment (mswertz) [inline apps/molgenis-components/src/components/forms/RowEdit.vue:259]: this seems okay to move it here.

**Comment by mswertz** [comment]
I think you made too many changes

File: apps/molgenis-components/src/components/forms/RowEdit.vue (hunk 1 above)
Comment (chinook25) [reply apps/molgenis-components/src/components/forms/RowEdit.vue:89]: A quick search through the entire application tells me it is not being used anywhere (I looked at where the RowEdit component was being imported).

**Review by chinook25 — APPROVED** [review]

File: apps/molgenis-components/src/components/forms/RowEdit.vue (hunk 3 above)
Comment (MaxPostema) [reply apps/molgenis-components/src/components/forms/RowEdit.vue:267]: This is done to trigger setting the default value only once, the old location triggered on every key press (setting back the original default value internally and not reflecting the reset in the user interface ).

**Review by mswertz — APPROVED** [review]

**Pull request merged**